scripts/rebuild-deb.sh

```diff
@@ -82,6 +82,46 @@ ls -lh mina*.deb
 
 ##################################### END GENERATE KEYPAIR PACKAGE #######################################
 
+##################################### SNAPP TEST TXN #######################################
+
+mkdir -p "${BUILDDIR}/DEBIAN"
+cat << EOF > "${BUILDDIR}/DEBIAN/control"
+
+Package: mina-snapp-test-transaction
+Version: ${MINA_DEB_VERSION}
+License: Apache-2.0
+Vendor: none
+Architecture: amd64
+Maintainer: O(1)Labs <build@o1labs.org>
+Installed-Size:
+Depends: ${SHARED_DEPS}
```

Comment:
This should be `Depends: ${SHARED_DEPS}${DAEMON_DEPS}` to take advantage of the logic near line ~30 which picks the appropriate libjemalloc and procps library for the given debian version